coreax/util.py

```diff
@@ -201,27 +202,39 @@ def solve_qp(kernel_mm: ArrayLike, kernel_matrix_row_sum_mean: ArrayLike) -> Arr
     return sol.primal
 
 
-def jit_test(fn: Callable, *args, **kwargs) -> tuple[float, float]:
+def jit_test(
+    fn: Callable,
+    fn_args: tuple = (),
+    fn_kwargs: dict | None = None,
+    jit_kwargs: dict | None = None,
+) -> tuple[float, float]:
     """
     Verify JIT performance by comparing timings of a before and after run of a function.
 
     The function is called with supplied arguments twice, and timed for each run. These
     timings are returned in a 2-tuple.
 
-    Note that `fn` often uses a lambda wrapper around a function or method call (see
-    performance tests) to ensure that the function or method is recompiled when called
-    multiple times, to truly test the JIT performance. In some cases, not doing this
-    will result in the re-use of previously cached information.
-
     :param fn: Function callable to test
     :return: (First run time, Second run time)
     """
+    # Avoid W0102
```

Comment:
Can you expand on this comment inside the codebase so its easier to come back to and understand in the future - what is W0102?